applications/roloi.md

```diff
@@ -0,0 +1,161 @@
+* Creation of a **UI template** for transaction automation
+* Extend template with transaction automation at a **smart contract level**
+
+**This will benefit the OAK ecosystem by providing scalable UI and smart contract templates that can be used by any team, while helping Roloi to set the foundations to integrate the recurring payments feature in the platform using OAK.**
+
+
+## Team 
+
+### Team members
+
+We are NeoPower, a Web3 software company. Our core team members:
+- **Brian Sasbon** (Co-Founder & CEO)
+- **Pablo Corrado** (Co-Founder & CTO)
+- **Hernán Hermida** (CGO)
+
+
+### Contact
+
+* **Contact Name:** Brian Sasbon
+* **Contact Email:** bsasbon@neopower.digital
+* **Website:** https://www.neopower.digital/ 
+
+### Legal Structure
+
+* **Registered Address:** Pacheco 2131, C1431, CABA, Buenos Aires, Argentina
+* **Registered Legal Entity:** NeoPower Digital, LLC
+
+### Team's experience
+
+#### Founders
+
+We are Brian and Pablo, **Software Engineers** with a Degree from the National Technological University (UTN) from Buenos Aires, Argentina, and have more than **10 years of experience** as developers for many different projects. We have strong experience working as **Full-stack Developers** and **Team Managers** for US traditional finance clients like Morgan Stanley, Visa and JP Morgan.
+
+6 years ago **we founded NeoPower**, our own software company to work for different clients building freelance teams of designers, developers and testers.
+
+We started our journey with NeoPower to solve real-world problems with Web3 products that are secure and have a great user experience. Roloi is our main product.
+
+
+#### CGO
+
+**Hernán S. Hermida** (aka [Milstein](https://twitter.com/MilsteinmAb)) is a **DeFi Researcher** with more than 2 years of experience contributing with educational communities in Latam. Currently, he **hosts #DeFiSpace** a weekly Twitter Spaces cycle, with more than 50 episodes, interviewing builders from different multi-chain projects, and creates content for [“La Multisig”](https://www.youtube.com/@LaMultisig), a web3 educational YouTube Channel with its own community. 
+
+He’s the growth lead of [DeFi Argentina](https://twitter.com/DeFiArgentina) a non-profit project that helps children’s food banks in Argentina by collecting donations in cryptocurrencies. 
+
+He’s also an **OAK ambassador** since Nov 2022.
+
+Hernán is part of the **Roloi team** as CGO to help with the growth, research and networking strategy, leveraging his knowledge about DeFi and communities.
+
+### Team Code Repos
+* [NeoPower Github Repository](https://github.com/NeoPower-Digital)
+
+## Development Status 
+
+We are working with the library [inkathon](https://github.com/scio-labs/inkathon) to simplify UI development and connection to the blockchain, this allows developers to focus more on solving actual problems.
```

Comment:
I am not familar with this project and don’t see it used by many yet. Do you have an example of a mature product that is built with this framework?

Reply:
This library provides some typesafe react hooks to leverage some abstractions over polkadot.js and simplify project bootstrapping.

The correct link is: [use-inkathon](https://github.com/scio-labs/use-inkathon)

We think it’s better to use something that’s being pushed by ecosystem teams like [Astar](https://twitter.com/AstarNetwork/status/1630553481307385865) and [ink!](https://twitter.com/ink_lang/status/1637765782644957186?s=20), over implementing custom abstractions.